.github/workflows/ci.yml

```diff
@@ -144,6 +144,13 @@ jobs:
             nodejs_hostname: nodejs.org
             nodejs_version: "18.17.0"
             nodejs_version_major: 18
+          - platform: linux-ppc64
+            distro: bullseye
+            run_on_arch: ppc64le
+            nodejs_arch: ppc64
```

Comment:
Does this need to be `ppc64le` to match the filename at https://nodejs.org/download/release/v18.17.0/ ?